core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpLeonardoDAO.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import org.broadinstitute.dsde.workbench.client.leonardo.api.AppsV2Api
+import org.broadinstitute.dsde.workbench.client.leonardo.model.CreateAppRequest
+import org.broadinstitute.dsde.workbench.client.leonardo.model.AppType
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.client.leonardo.ApiClient
+
+class HttpLeonardoDAO(
+  baseUrl: String,
+  wdsType: String
+) extends LeonardoDAO {
+
+  def getAppsV2leonardoApi(accessToken: String): AppsV2Api = {
+    val apiClient = new ApiClient()
+    apiClient.setAccessToken(accessToken)
+    apiClient.setBasePath(baseUrl)
+    new AppsV2Api(apiClient)
+  }
+
+  def createWDSInstance(token: String, workspaceId: UUID, appName: String): Unit = {
+    createApp(token, workspaceId, appName, wdsType);
+  }
```

Comment:
I suggest moving the knowledge of how to name the WDS app into here, instead of keeping it in `MultiCloudWorkspaceService`:
```scala
  def createWDSInstance(token: String, workspaceId: UUID): Unit = {
    createApp(token, workspaceId, s"wds-${workspaceId}", wdsType);
  }
```
Trying to keep all the WDS-specific logic in one place (such as "CROMWELL" vs "WDS" for the app type).